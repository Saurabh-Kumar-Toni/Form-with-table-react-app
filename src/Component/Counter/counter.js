import React, {  } from 'react';

const Counter = ({count}) => {
    return( 
        <div id ="counter">
            <p>
                Our Total User Count Is: {count}
            </p>
            <p> Add more to increase the number</p>
        </div>
    );
}

export default Counter;