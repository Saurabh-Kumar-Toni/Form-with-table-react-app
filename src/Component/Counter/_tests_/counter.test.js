import React from 'react';
import { render } from '@testing-library/react';
import Counter from '../counter'


test('Check if Counter Paragraph with value is rendered', () => {
   
    const counts = 2;
    const {container} = render(<Counter count={counts}><p>
        Our Total User Count Is: {counts}
        </p></Counter>);
    expect(container.firstChild).toBeInTheDocument();
  });

  test('Check if counter component is renedered or not', () => {
 
    const counts = 2;
    const {container} = render(<Counter count={counts} />)
    expect(container.firstChild).toMatchSnapshot();
  });
  