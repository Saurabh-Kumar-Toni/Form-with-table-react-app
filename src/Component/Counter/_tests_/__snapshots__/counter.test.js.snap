// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if counter component is renedered or not 1`] = `
<div
  id="counter"
>
  <p>
    Our Total User Count Is: 
    2
  </p>
  <p>
     Add more to increase the number
  </p>
</div>
`;
